generate-color-css.js

```diff
@@ -1,337 +1,829 @@
+init_pair(59, green, magenta);
+init_pair(60, yellow, magenta);
+init_pair(61, blue, magenta);
+init_pair(62, magenta, magenta);
+init_pair(63, cyan, magenta);
 
-    init_pair( 64, black,      cyan );
-    init_pair( 65, white,      cyan );
-    init_pair( 66, red,        cyan );
-    init_pair( 67, green,      cyan );
-    init_pair( 68, yellow,     cyan );
-    init_pair( 69, blue,       cyan );
-    init_pair( 70, magenta,    cyan );
-    init_pair( 71, cyan,       cyan );
+init_pair(64, black, cyan);
+init_pair(65, white, cyan);
+init_pair(66, red, cyan);
+init_pair(67, green, cyan);
+init_pair(68, yellow, cyan);
+init_pair(69, blue, cyan);
+init_pair(70, magenta, cyan);
+init_pair(71, cyan, cyan);
 
 function color_pair(i) {
   return {
     fg: color_pairs[i][0],
     bg: color_pairs[i][1],
-    bold() { return {...this, is_bold: true} },
-    blink() { return {...this, is_blink: true} },
+    bold() {
+      return { ...this, is_bold: true };
+    },
+    blink() {
+      return { ...this, is_blink: true };
+    },
     is_bold: false,
     is_blink: false,
     color() {
-
       let fg = this.fg;
       let bg = this.bg;
-      if (this.is_bold)
-        fg = fg + 8;
-      if (this.is_blink)
-        bg = bg + 8;
-      return { fg, bg }
-    }
-  }
+      if (this.is_bold) fg = fg + 8;
+      if (this.is_blink) bg = bg + 8;
+      return { fg, bg };
+    },
+  };
 }
 
-all_colors = {}
+all_colors = {};
 function add_color(def, name, pair, inverse) {
   all_colors[name] = {
     ...pair.color(),
-    inverse
-  }
+    inverse,
+  };
 }
 
-    add_color( "def_c_black", "c_black", color_pair( 30 ), "def_i_black" );
-    add_color( "def_c_white", "c_white", color_pair( 1 ).bold(), "def_i_white" );
-    add_color( "def_c_light_gray", "c_light_gray", color_pair( 1 ), "def_i_light_gray" );
-    add_color( "def_c_dark_gray", "c_dark_gray", color_pair( 30 ).bold(), "def_i_dark_gray" );
-    add_color( "def_c_red", "c_red", color_pair( 2 ), "def_i_red" );
-    add_color( "def_c_green", "c_green", color_pair( 3 ), "def_i_green" );
-    add_color( "def_c_blue", "c_blue", color_pair( 4 ), "def_i_blue" );
-    add_color( "def_c_cyan", "c_cyan", color_pair( 5 ), "def_i_cyan" );
-    add_color( "def_c_magenta", "c_magenta", color_pair( 6 ), "def_i_magenta" );
-    add_color( "def_c_brown", "c_brown", color_pair( 7 ), "def_i_brown" );
-    add_color( "def_c_light_red", "c_light_red", color_pair( 2 ).bold(), "def_i_light_red" );
-    add_color( "def_c_light_green", "c_light_green", color_pair( 3 ).bold(), "def_i_light_green" );
-    add_color( "def_c_light_blue", "c_light_blue", color_pair( 4 ).bold(), "def_i_light_blue" );
-    add_color( "def_c_light_cyan", "c_light_cyan", color_pair( 5 ).bold(), "def_i_light_cyan" );
-    add_color( "def_c_pink", "c_pink", color_pair( 6 ).bold(), "def_i_pink" );
-    add_color( "def_c_yellow", "c_yellow", color_pair( 7 ).bold(), "def_i_yellow" );
+add_color("def_c_black", "c_black", color_pair(30), "def_i_black");
```

Comment:
can we prettier-ignore this block? it's copy/pasted and tweaked from src/color.cpp: https://github.com/CleverRaven/Cataclysm-DDA/blob/d978eeb966150794689ea1216d18f00828abc683/src/color.cpp#L214